docs/source/conf.py

```diff
@@ -0,0 +1,71 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('../../db-facts/'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'db-facts'
+copyright = '2020 BlueLabs Analytics, Inc.'
+author = 'Vince Broz'
+
+# The full version, including alpha/beta/rc tags
+release = '4.0.0'
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    # The default sphinx doesn't seem to set TYPE_CHECKING, which
+    # results in dropping our use of TypedDict, Literal and other
+    # things we don't use by default to avoid extra dependencies.
+    'sphinx_autodoc_typehints',
```

Comment:
The net result from `sphinx_autodoc_typehints` could be better - but it's definitely better than seeing the detailed return value from `db()` as just `dict`.